src/gpu_sysman.c

```diff
@@ -249,6 +249,54 @@ static int gpu_config_free(void) {
   return RET_OK;
 }
 
+/* list GPU metric options that can get disabled at run-time */
+static unsigned int list_gpu_metrics(const gpu_disable_t *disabled) {
+  struct {
+    bool disabled;
+    bool submetric;
+    const char *name;
+  } names[] = {{disabled->engine, false, "Engine"},
+               {disabled->fabric, false, "Fabric port"},
+               {disabled->freq, false, "Frequency"},
+               {disabled->mem, false, "Memory"},
+               {disabled->membw, false, "Memory BW"},
+               {disabled->power, false, "Power"},
+               {disabled->power_ratio, true, "Power ratio"},
+               {disabled->ras, false, "RAS/errors"},
+               {disabled->temp, false, "Temperature"},
+               {disabled->throttle, false, "Throttle time"}};
+
+  unsigned int i;
+
+  if (config.gpuinfo) {
+    unsigned int disabled = 0;
+    INFO("Disabled metrics / submetrics:");
+    for (i = 0; i < STATIC_ARRAY_SIZE(names); i++) {
+      if (names[i].disabled) {
+        INFO("- %s%s", names[i].name, names[i].submetric ? " (submetric)" : "");
+        disabled++;
+      }
+    }
+    if (!disabled) {
+      INFO("- none");
+    }
+    INFO("Enabled metrics:");
```

Comment:
Please move this out of this `if` block. The enabled metrics are printed unconditionally, and so should this line.

Reply:
All metrics printing *is* conditional on `config.gpuinfo`.  But I moved it out, as code is clearer that way.